docs/pages/index.mdx

```diff
@@ -40,8 +40,7 @@ be happy to help.
         width={416}
         height={234}
         quality={90}
-        layout="responsive"
-        objectFit="cover"
+        style={{objectFit:"contain"}}
```

Comment:
In Next.js 13 `objectFit` and `layout` are deprecated from the Image component and must be implemented using `style` or `className`. See https://nextjs.org/docs/app/building-your-application/upgrading/codemods#next-image-experimental